client/src/services/WebPlaybackService.js

```diff
@@ -0,0 +1,82 @@
+import BaseService from './BaseService'
+import SpotifyService from './SpotifyService'
+
+const play = ({
```

Comment:
Got this from https://beta.developer.spotify.com/documentation/web-playback-sdk/reference/#playing-a-spotify-uri